### PR TITLE
Laboratory work #1, Nushkina Ekaterina - 21FPL1

### PR DESCRIPTION
**Pull request by eanushkina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Лабораторные работы для 2-го курса ФПЛ (2022/2023)
+git remote add upstream# Лабораторные работы для 2-го курса ФПЛ (2022/2023)
```
Comment (marina-kaz) [inline README.md:1]: Please, remove any changes in files other than main.py and start.py

**Review by marina-kaz — COMMENTED** [review]
Nothing to review yet :(
Hurry up, the deadline is approaching!

**Review by marina-kaz — COMMENTED** [review]
Nothing to review yet :(
Consider catching up as deadline is approaching

**Review by marina-kaz — COMMENTED** [review]
I'm afraid you won't be able to finish the lab in time :( 
Hope to see some improvements next week!

**Review by marina-kaz — COMMENTED** [review]
Nothing to review just yet :(

**Review by marina-kaz — COMMENTED** [review]
Nothing to review just yet :(

**Review by marina-kaz — COMMENTED** [review]
Nothing to review so far :(

**Review by marina-kaz — COMMENTED** [review]
Nothing to review :<

**Review by marina-kaz — COMMENTED** [review]
No changes since the last review

**Review by marina-kaz — COMMENTED** [review]
Nothing to review today :(

**Review by marina-kaz — COMMENTED** [review]
I think you've pushed your changes before pulling the existing ones
Let me know if you need help restoring your repo to the workable state

**Comment by eanushkina** [comment]
yes, i need your help, please

**Review by marina-kaz — COMMENTED** [review]
No new changes :<

**Review by marina-kaz — COMMENTED** [review]
No new changes to review :<

**Pull request closed without merging**